### PR TITLE
[WIP] Send errors to our beacon

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: config/bugsnag.php [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    | Enable this if you'd like us to set the current user logged in via
+    | Laravel's authentication system.
+    |
+    | If you'd like to add your own user resolver, you can call the
+    | Bugsnag::registerUserResolver method from the boot method of your app
+    | service provider.
+    |
+    */
+
+    'user' => true,
```
Comment (jbrooksuk) [inline config/bugsnag.php]: I don't think we should do this. Since users from several status pages would be grouped into one on Bugsnag.

Comment (GrahamCampbell) [inline config/bugsnag.php]: Not if we transform it on our end.

Comment (GrahamCampbell) [inline config/bugsnag.php]: That said, maybe user information is private, and we shouldn't send it to us?

Comment (jbrooksuk) [inline config/bugsnag.php]: I don't think we should be sending it to us. Beacon is anonymous on a whole, with the exception being the support contact email.

**Pull request closed without merging**